source/upgrade.txt

```diff
@@ -29,6 +29,16 @@ Before you upgrade, perform the following actions:
   more about the {+mdb-server+} release compatibility changes, see the 
   :ref:`<node-server-support-changes>` section.
 
+After completing the preceding steps, navigate to your project directory in your 
```

Comment:
Issue:

I think the information is clear and precise, but also that this content is out of place. It seems like it is providing a step-by-step "how-to" guide on upgrading, but is located in the "Overview" of the page. Our team uses the Overview section to define terminology and summarize what information can be found on the page.

I think the "Before you upgrade" content that is currently under "Overview" does not belong, and may have been confusing.

Suggestion:

1) I think it could be helpful to create a new section called "How to Upgrade" and move all the relevant content from the "Overview" section to it.

2) I think it would be helpful to modify the content you added to present the information without referring to procedural (step-by-step) instructions. E.g. "To upgrade your driver version, run the following command in your application's directory:"